ansible/roles/davros/tasks/authproxy.yml

```diff
@@ -5,6 +5,14 @@
     src: authproxy/
     dest: /opt/davros/authproxy/
 
+- name: Set up file permissions on authproxy folder
+  file:
+    path: /opt/davros/authproxy
+    state: directory
+    owner: liquid
```

Comment:
Why does liquid need to write in this folder?

Reply:
Because I have set up the oauth keys with permission for the `liquid` user, and that means it needs those permissions to write them here.

It would be simpler to have all those keys set up as root, because they're only written at first boot.

Reply:
So let's do it as root then.